src/plugins/intel_gna/src/transformations/insert_copy_layer.cpp

```diff
@@ -115,6 +116,43 @@ InsertCopyBeforeConcatLayer::InsertCopyBeforeConcatLayer() {
     this->register_matcher(m, callback);
 }
 
+InsertCopyBeforeBroadcastLayer::InsertCopyBeforeBroadcastLayer() {
+    MATCHER_SCOPE(InsertCopyBeforeBroadcastLayer);
+
+    auto broadcast_op = ngraph::pattern::wrap_type<ngraph::opset9::Broadcast>();
```

Comment:
Lets start using the new namespaces ov:
using namespace ov::opset9;
....
auto broadcast_op = ngraph::pattern::wrap_type<Broadcast>();

instead of the old ngraph namespaces

Reply:
done